build-tools/packages/build-infrastructure/README.md

```diff
@@ -0,0 +1,235 @@
+    // This is the name of the workspace which is how it's referenced in the API.
+    "client": {
+      // This workspace is rooted at the root of the Git repo.
+      directory: ".",
+      releaseGroups: {
+        // This key is the name of the release group. It must be unique across all
+        // release groups.
+        client: {
+          // The include property can contain package names OR package scopes. If
+          // a scope is provided, all packages with that scope will be a part of
+          // the release group.
+          include: [
+            // Include all the Fluid Framework scopes and packages except for
+            // @fluid-example.
+            "@fluidframework",
+            "@fluid-experimental",
+            "@fluid-internal",
+            "@fluid-private",
+            "@fluid-tools",
+            "@types/jest-environment-puppeteer"
+            "fluid-framework",
+          ],
+          // A release group can have an OPTIONAL root package. This package
+          // is typically private and is similar to the root package for a workspace.
+          // This release group root package may be useful to store scripts or other
+          // configuration only applies on the release group,
+          rootPackageName: "client-release-group-root",
+          defaultInterdependencyRange: "workspace:~",
+
+          // A release group may have an ADO pipeline URL associated with it. This
+          // URL is used to provide direct links to the pipeline when running releases.
+          adoPipelineUrl:
+            "https://dev.azure.com/fluidframework/internal/_build?definitionId=12",
+        },
+        examples: {
+          // This release group contains only the @fluid-example packages.
+          include: ["@fluid-example"],
+          rootPackageName: "examples-release-group-root",
+          defaultInterdependencyRange: "workspace:~",
+        },
+        // If any packages in the workspace don't match a release group, loading the
+        // repo layout config will throw an error.
+      },
+    },
+    "build-tools": {
+      // This workspace is rooted in the build-tools folder. This folder must contain
+      // a workspace config. The specific config depends on the package manager being used.
+      directory: "./build-tools",
+      releaseGroups: {
+        // Release groups can have the same name as workspaces, but all release group names
+        // must be unique regardless of the workspace they belong to.
+        "build-tools": {
+          include: [
+            // Include all Fluid Framework scopes. Only packages contained in the workspace
+            // will be included, so it is safe to use the same scopes in multiple release
+            // group definitions as long as they're in different workspaces.
+            "@fluidframework",
+            "@fluid-example",
+            "@fluid-experimental",
+            "@fluid-internal",
+            "@fluid-private",
+            "@fluid-tools",
```

Comment:
Clean up this one to only have the ones actually in build-tools.

Reply:
This is intentional: "Only packages contained in the workspace will be included, so it is safe to use the same scopes in
multiple release group definitions as long as they're in different workspaces."